site/docs/red-team/configuration.md

```
@@ -148,6 +150,69 @@ plugins:
+redteam:
+  testGenerationInstructions: |
+    Focus on healthcare-specific attacks using medical terminology and patient scenarios. 
+    Ensure all prompts reference realistic medical situations that could occur in patient interactions.
+    Consider HIPAA compliance requirements when generating privacy-related attacks.
+```
+
+#### How Test Generation Instructions Work
+
+When specified, these instructions are automatically combined with each plugin's base template using the following format:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)